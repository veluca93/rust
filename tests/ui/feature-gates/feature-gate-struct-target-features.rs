#[target_feature(enable = "avx")] //~ ERROR attribute should be applied to a function definition
struct Avx {}

fn main() {}
