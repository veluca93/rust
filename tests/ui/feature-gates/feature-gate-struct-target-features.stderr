error: attribute should be applied to a function definition
  --> $DIR/feature-gate-struct-target-features.rs:1:1
   |
LL | #[target_feature(enable = "avx")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
LL | struct Avx {}
   | ------------- not a function definition

error: aborting due to 1 previous error

